### PR TITLE
User post review

### PR DESCRIPTION
**Pull request by tinobia**

### CODE REVIEW

**Comment by tinobia** [comment]
ready

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class ReviewsController < ApplicationController
+  def create
+    @review = current_user.reviews.new review_params
+    @tour = Tour.find_by id: params[:tour_id]
+    unless @tour && @review.save && @review
+      flash[:danger] = t "controllers.reviews_controller.not_success"
+    end
+    redirect_to tour_path(@tour)
```
Comment (dothidiemthao) [inline app/controllers/reviews_controller.rb]: bỏ ( )

File: app/controllers/tourdetails_controller.rb [hunk 2]
```diff
@@ -35,7 +35,7 @@ def find_tour tour_id
   end
 
   def find_tour_detail
-    return if @tourdetail = Tourdetail.find(params[:id])
+    return if @tourdetail == Tourdetail.find(params[:id])
```
Comment (dothidiemthao) [inline app/controllers/tourdetails_controller.rb]: dùng find by

**Comment by dothidiemthao** [comment]
commented

**Comment by tinobia** [comment]
ready

**Comment by tinobia** [comment]
ready

**Pull request merged**